prelude-si/rootfs/rootfs_build.sh

```diff
@@ -178,6 +184,13 @@ EOZ
 EOL
 # LEAVE CHROOT
 
+# See https://github.com/denoland/deno_docker/blob/main/alpine.dockerfile for
+# the inspiration here. We get the glibc bits directly from distroless/cc
```

Comment:
Oh, I see, this build/linking issue is related to Deno specifically (at least in part)! It was these kinds of challenges that had me drop the use of the `pkg` npm package--I just could not get the library paths to work correctly.